Dockerfile.template

```diff
@@ -76,7 +77,10 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 	&& \
 	# we can change from world to world-bin in newer releases
 	make world && \
-	make install-world
+	make install-world \
+	&& \
+	curl -sSL "https://github.com/pgpartman/pg_partman/archive/v${PARTMAN_VERSION}.tar.gz" | tar -xz && \
```

Comment:
if i recall correctly, this run step takes the longest and would probably benefit the most from being cached for faster builds. i think separating each extension into it's own layer would also help with potential debugging